agents/birdwatcher/tinybird/endpoints/get_latest_slack_oauth_tokens.pipe

```diff
@@ -0,0 +1,17 @@
+DESCRIPTION >
+    Get the latest Slack OAuth tokens for a specific team_id.
+
+NODE get_latest_slack_oauth_tokens_node
+SQL >
+    SELECT
+        team_id,
+        bot_token,
+        bot_user_id,
+        authed_user_id,
+        updated_at
+    FROM slack_oauth_tokens
```

Comment:
not needed but let's do `FROM slack_oauth_tokens FINAL` here to remember this is a RMT 🙏